### PR TITLE
feat: update colors to be consistent with neovim and wezterm

### PR DESCRIPTION
**Pull request by nshern**
In relation to issue #6 I updated some of the colors. See the the screenshot for a comparison.

Screenshot 2023-11-12 at 23 10 54

### CODE REVIEW

**Comment by nshern** [comment]
@mvllow would you prefer that I split the changes up into individual PR's so they can be assessed individually?

**Comment by nshern** [comment]
Changed bufferline colors to be more legible and consistent with [wezterm tab colors](https://github.com/neapsix/wezterm)
Screenshot 2023-11-13 at 14 45 48

**Comment by LeoniePhiline** [comment]
Hi @nshern 

I find it a regression that module paths are no longer easily differentiated from local variables. (Both are white / foam now.)

Could you please consider re-coloring paths? (i.e. `math` in your screenshot above)

I would imagine neovim / vscode users would also benefit from that.

**Comment by nshern** [comment]
> Hi @nshern
> 
> I find it a regression that module paths are no longer easily differentiated from local variables. (Both are white / foam now.)
> 
> Could you please consider re-coloring paths? (i.e. `math` in your screenshot above)
> 
> I would imagine neovim / vscode users would also benefit from that.

Hi @LeoniePhiline,

Thank you for the feedback. I have simply made the change to correspond to, what I understand to be, the canonical version of the theme (VS Code) and do not feel qualified to make decisions on how the theme should be applied.

Perhaps @mvllow  can offer some input?

**Comment by mvllow** [comment]
After some digging, it appears that Neovim and Helix simply use different groups for their highlighting.

**Neovim vs Helix**

```diff
- import (Include) math (@variable)
+ import (keyword) math (namespace)
```

Perhaps making them visually consistent is not the best solution. Both use tree sitter so I'm surprised they're different.

**Comment by flinesse** [comment]
I would also consider this a regression for my use case, although helix makes it easy for me to provide my own overrides. 

As @LeoniePhiline mentioned—in Rust, module paths (`namespace`) become indiscernible from local variables and struct field labels (`variable.other.member`)  blend into its type. I imagine this is the case for other C-family languages.

It's worth noting that there will be variations in highlighting scopes and the language demographics defining them across different editors. Thus, it may be wiser to defer to individual user groups in finding areas of incremental improvement rather than to try and enforce visual consistency from the "canonical version".

**Comment by mvllow** [comment]
I appreciate the feedback here, but it a little hard to follow in a closed PR. Please see #10. I am happy to make or accept any changes but would love if we can gather actionable items there. Thank you :)
